lib/matplotlib/cbook.py

```diff
@@ -566,6 +566,18 @@ def process(self, s, *args, **kwargs):
                     self._remove_proxy(proxy)
 
 
+class EventEmitter(object):
+    def __init__(self):
+        super(EventEmitter, self).__init__()
```

Comment:
Maybe add a brief comment as to why `super` (and any base `__init__`) is needed here.  I had the idea that it was *not* needed for a class that inherits only from `object`.